src/email.js

```diff
@@ -0,0 +1,41 @@
+import debug from 'debug';
+import nodemailer from 'nodemailer';
+import EmailError from './errors';
+
+const log = debug('uwave:api:v1:email');
+
+export function sendEmail(emailAddress, token, options) {
+  const smtpOptions = {
+    host: 'localhost',
+    port: 25,
+    debug: true,
+    tls: {
+      rejectUnauthorized: false,
+    },
+  };
+
+  var mailTransport = null;
+  if (options.mailTransport) {
+    mailTransport = options.mailTransport;
+  } else {
+    mailTransport = nodemailer;
+  }
+
+  // create reusable transporter object using the default SMTP transport
+  const transporter = mailTransport.createTransport(smtpOptions);
```

Comment:
Custom nodemailer transports work by passing the transport to `nodemailer.createTransport`. So this line should be `nodemailer.createTransport(options.mailTransport || smtpOptions)`, in order to use the custom transport if the option is specified, or else the default smtp transport.